controllers/admin_domains.php

```diff
@@ -158,6 +158,9 @@ function ($service1, $service2) use ($alt_sort, $order) {
         );
         $this->setPagination($this->get, $settings);
 
+        // Set the page title
+        $this->structure->set('page_title', Language::_('AdminDomains.browse.page_title', true));
```

Comment:
Most of these individual calls to set the page title are unnecessary, this is taken care of for the admin interface in AppController as long as the language key follows the format Controller.action.page_title.  The call for the client interface is still necessary though.